enterprise_access/apps/api/v1/tests/test_bff_views.py

```diff
@@ -352,73 +352,142 @@ def test_dashboard_with_subscriptions_license_activation(
         {
+            'has_existing_activated_license': False,
+            'has_existing_revoked_license': False,
+            'is_staff_request_user': False,
             'identity_provider': False,
             'auto_apply_with_universal_link': False,
             'has_plan_for_auto_apply': True,
             'should_auto_apply': False,
         },
-        # Identity provider exists, no universal link auto-apply, but a plan for auto-apply exists.
+        # No existing licenses, identity provider exists, no universal link auto-apply,
+        # a plan for auto-apply exists, and not a staff request user.
         # Expected: Should auto-apply.
         {
+            'has_existing_activated_license': False,
+            'has_existing_revoked_license': False,
+            'is_staff_request_user': False,
             'identity_provider': True,
             'auto_apply_with_universal_link': False,
             'has_plan_for_auto_apply': True,
             'should_auto_apply': True,
         },
-        # No identity provider, universal link auto-apply is enabled, and a plan for auto-apply exists.
+        # No existing licenses, no identity provider, universal link auto-apply is enabled,
+        # a plan for auto-apply exists, and not a staff request user.
         # Expected: Should auto-apply.
         {
+            'has_existing_activated_license': False,
+            'has_existing_revoked_license': False,
+            'is_staff_request_user': False,
             'identity_provider': False,
             'auto_apply_with_universal_link': True,
             'has_plan_for_auto_apply': True,
             'should_auto_apply': True,
         },
-        # Identity provider exists, universal link auto-apply is enabled, and a plan for auto-apply exists.
+        # No existing licenses, identity provider exists, universal link auto-apply is enabled,
+        # a plan for auto-apply exists, and not a staff request user.
         # Expected: Should auto-apply.
         {
+            'has_existing_activated_license': False,
+            'has_existing_revoked_license': False,
+            'is_staff_request_user': False,
             'identity_provider': True,
             'auto_apply_with_universal_link': True,
             'has_plan_for_auto_apply': True,
             'should_auto_apply': True,
         },
+        # Existing activated license, identity provider exists, no universal link auto-apply,
+        # a plan for auto-apply exists, and not a staff request user.
+        # Expected: Should not auto-apply.
+        {
+            'has_existing_activated_license': True,
+            'has_existing_revoked_license': False,
+            'is_staff_request_user': False,
+            'identity_provider': True,
+            'auto_apply_with_universal_link': True,
+            'has_plan_for_auto_apply': True,
+            'should_auto_apply': False,
+        },
+        # Existing revoked license, identity provider exists, no universal link auto-apply,
```

Comment:
these say "no universal link auto-apply", but the data for both has `'auto_apply_with_universal_link': True,` - is there a test permutation we're missing? or maybe just copypasta in the comment?

Reply:
D'oh, just copy/pasta error. Will fix!